warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -179,10 +200,23 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_details = selenium_Utils.\
                     get_browser_details(browser, datafile=self.datafile, **arguments)
             if browser_details is not None:
+                if str(headless_mode).strip().lower() == "yes":
```

Comment:
Hi @Annam16,

Instead of a direct ```==```, can we have a ReGex that supports "y" and "yes" (case insensitive)? Please also update the docstrings accordingly.